### PR TITLE
i18n(es): Update `view-transitions.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Update `view-transitions.mdx` [`84a5711`](https://github.com/withastro/docs/commit/84a5711f0fce75714cf0a557e44e03861ff4f586) & [`b842956`](https://github.com/withastro/docs/commit/b842956d6c4abaa7510f5d4723f1f223fd074fb7)

### CODE REVIEW

File: src/content/docs/es/guides/view-transitions.mdx [hunk 1]
```diff
@@ -233,21 +233,86 @@ const myFade = {
...
 
-```html
-<a href="/ganancias-trimestrales.pdf" data-astro-reload>
+Las siguientes opciones te permiten controlar aún más cuándo ocurre la navegación dentro del enrutador:
+
+- `astro-data-reload`: un atributo de la etiqueta `<a>` para [forzar una navegación de página completa](#evitar-la-navegación-del-lado-del-cliente).
+- `astro-data-history="auto | push | replace"`: un atributo de la etiqueta `<a>` para [controlar el historial del navegador](#reemplazar-entradas-en-el-historial-del-navegador).
+- `navigate(href, options)`: un método disponible para cualquier script o componente cliente para [desencadenar la navegación](#desencadenar-la-navegación).
+
+### Evitar la navegación del lado del cliente
```
Comment (Waxer59) [inline src/content/docs/es/guides/view-transitions.mdx:249]:
```suggestion
### Previniendo la navegación del lado del cliente
```

File: src/content/docs/es/guides/view-transitions.mdx [hunk 2]
```diff
@@ -233,21 +233,86 @@ const myFade = {
...
 
-En estos casos, puedes optar por no utilizar el enrutamiento del lado del cliente en cada enlace individual mediante el atributo `data-astro-reload`, de la siguiente manera:
+- Los clics en elementos `<a>`.
+- Eventos de navegación hacia atrás y hacia adelante.
 
-```html
-<a href="/ganancias-trimestrales.pdf" data-astro-reload>
+Las siguientes opciones te permiten controlar aún más cuándo ocurre la navegación dentro del enrutador:
+
+- `astro-data-reload`: un atributo de la etiqueta `<a>` para [forzar una navegación de página completa](#evitar-la-navegación-del-lado-del-cliente).
```
Comment (Waxer59) [inline src/content/docs/es/guides/view-transitions.mdx:245]:
```suggestion
- `astro-data-reload`: un atributo de la etiqueta `<a>` para [forzar una navegación de página completa](#previniendo-la-navegación-del-lado-del-cliente).
```

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**